core/modules/views/includes/view.inc

```diff
@@ -49,12 +50,6 @@ class view {
    */
   var $human_name;
 
-  /**
-   * The core version the view was created for.
-   * @var int
-   */
-  var $core;
```

Comment:
This is basically the only substantial change in this file; the rest is just fixing comments.